app/_hub/kong-inc/kafka-upstream/index.md

```diff
@@ -1,6 +1,8 @@
 ---
 name: Kafka Upstream
 publisher: Kong Inc.
+version: 1.3-x
+# internal 0.0.2, only former version 0.0.1.
```

Comment:
didn't get this part

`internal 0.0.2, only former version 0.0.1.`

same question from #2595 applies

Reply:
This is more of a note to self that there are plugin version issues, and that updates did not happen as they should have in the past. You'll note there is only an index file, no other versions. Need to discuss this with @eskibars and come up with a solution as to how we present and update plugin versions. Confluent only presents the most current version of a connector on their hub; perhaps we should do the same.